.ibm/pipelines/value_files/values_showcase-rbac.yaml

```
@@ -205,7 +205,7 @@ upstream:
       - name: install-dynamic-plugins
         # -- Image used by the initContainer to install dynamic plugins into the `dynamic-plugins-root` volume mount.
         # It could be replaced by a custom image based on this one.
-        # @default -- `quay.io/janus-idp/backstage-showcase:latest`
+        # @default -- `quay.io/rhdh-community/rhdh:latest`
```

There is no :latest tag, only :next 

```suggestion
        # @default -- `quay.io/rhdh-community/rhdh:next`
```